### PR TITLE
Removing "any" constraints and doing some minor cleanup. 

### PR DESCRIPTION
**Pull request by Sfshaza**
Have verified that all examples run both in Dartium and browser.

@kwalrath

### CODE REVIEW

File: todo_with_delete/pubspec.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 name: todo_with_delete
 description: A sample application
 dependencies:
-  browser: any
+  browser: ^0.10.0+2
```
Comment (kwalrath) [inline todo_with_delete/pubspec.yaml]: `^0.10.0+2` -> `^0.10.0`
? (GLOBAL)

File: todo_with_delete/web/todo_with_delete.html [hunk 2]
```diff
@@ -26,6 +26,6 @@
     <button id="delete-all" type="button" style="float:right"> Delete All </button>
     
     <script type="application/dart" src="todo_with_delete.dart"></script>
-    <script src="packages/browser/dart.js"></script>
+    <script type="text/javascript" src="packages/browser/dart.js"></script>
```
Comment (kwalrath) [inline todo_with_delete/web/todo_with_delete.html]: delete `type="text/javascript"` (global)

File: portmanteaux_simple/web/portmanteaux_simple.html [hunk 3]
```diff
@@ -20,6 +20,6 @@
     </ul>
 
     <script type="application/dart" src="portmanteaux_simple.dart"></script>
-    <!-- for this next line to work, your pubspec.yaml file must have a dependency on 'browser' -->
```
Comment (kwalrath) [inline portmanteaux_simple/web/portmanteaux_simple.html]: why was this comment here?

Comment (Sfshaza) [inline portmanteaux_simple/web/portmanteaux_simple.html]: I don't know! It seemed so ancient and aged, I took it out.

**Comment by kwalrath** [comment]
LGTM!

**Pull request merged**